pkg/normalize/normalize.go

```diff
@@ -0,0 +1,55 @@
+package normalize
```

Comment:
Copied over from kubectl-gs

Reply:
Can we just call `kubectl-gs` as a binary in here? Like download `kubectl-gs` in the `standup` image?

Or alternatively call the kubectl-gs commands in code directly?

Reply:
I prefer having a different template for tests. I will try to use kubectl-gs' function though.

Reply:
done

Reply:
I think repeating the templates here instead of dogfooding kubectl-gs is not the way to go. I experimented with that approach like 1 year ago, but never got anywhere and the `standup` branch got removed :cry: All I have left is this tekton task I accidentally committed https://github.com/giantswarm/test-infra/blob/master/tekton/tasks/kubectl-gs.yaml

The idea was basically use kubectl-gs to generate the manifests and apply them from a tekton task. Using `kubectl-gs` parameters we could customize the manifests to whatever we want during tests.

Reply:
Okay, 2 against 1, you won.